pkg/sidecar-controller/framework_test.go

```diff
@@ -521,6 +522,7 @@ func newTestController(kubeClient kubernetes.Interface, clientset clientset.Inte
 		60*time.Second,
 		"snapshot",
 		-1,
+		workqueue.DefaultControllerRateLimiter(),
```

Comment:
why this is not changed to test the new rater limiter?

Reply:
Can you address this comment?

Reply:
done.. thanks ! @xing-yang ptal.